src/twisted/protocols/_smb/__init__.py

```diff
@@ -0,0 +1,10 @@
+# -*- test-case-name: twisted.protocols._smb.tests -*-
+# Copyright (c) Twisted Matrix Laboratories.
+# See LICENSE for details.
+"""
+Server Message Block protocol implementation
+"""
+
+from twisted.protocols._smb.core import SMBFactory
+
+__all__ = ['SMBFactory']
```

Comment:
Why is just this one name promoted up to the package namespace?

Reply:
simply so the user can access as twisted.protocols.smb.SMBFactory as opposed to twisted.protocols.smb.core.SMBFactory  Over time other classes required for public API (mainly interfaces) would be added.